### PR TITLE
RP Landing Page Workshop Wording Updates

### PR DESCRIPTION
**Pull request by TurnerRiley**
If a teacher is viewing available PD workshops based on their regional partner, they are currently shown an entry for available workshops of each type and (if CSA is not being offered) an entry stating CSA is not being offered:
c782f8f0-9791-433b-8491-becb920e3e77

The goal of this PR is to provide more details on what is being offered. If the Regional Partner is offering a workshop but has not scheduled it yet, the Regional Partner landing page will not notify the user that the workshop details will be coming soon. And if the Regional Partner is not offering a given course (not just CSA), the user will be notified of that as well. Here are some different scenarios to show the 3 versions of a workshop entry (offered and scheduled, offered but not scheduled, and not offered):

### RP offering CSD, CSP, and CSA with scheduled workshops for all 3
AllScheduled

### RP offering CSD and CSP but has only scheduled CSD
Offer2Schedule1

### RP offering only CSD but has not scheduled it yet
JustCSD

### RP is not offering any workshops at this time
NoneOffered

## Links
Spec: [here](https://docs.google.com/document/d/19jnaa1_9WYZ32tY6IGrrHmHTTYcao9cRkDzeHWq3-8I/edit#heading=h.3yj978pk57yq)
Jira ticket: [here](https://codedotorg.atlassian.net/browse/ACQ-179?atlOrigin=eyJpIjoiMjgwYzc4MWUwYWNmNDkwOGIyOTg2YTVkYjcyM2FkMDIiLCJwIjoiaiJ9)

## Testing story
Local testing and drone build passing.

## PR Checklist:
- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/RegionalPartnerSearch.jsx [hunk 1]
```diff
@@ -342,17 +343,38 @@ class RegionalPartnerSearch extends Component {
...
+                  } else if (collection.workshops.length > 0) {
+                    // If workshops present for the given course
+                    return (
+                      <div
+                        key={collectionIndex}
+                        style={{
+                          ...styles.workshopCollection,
+                          ...workshopCollectionStyle
+                        }}
+                      >
```
Comment (megcrenshaw) [inline apps/src/templates/RegionalPartnerSearch.jsx]: Nit: Thoughts about extracting this div since it's used three times?

**Review by megcrenshaw — APPROVED** [review]
Looks great! Thanks for getting this done

File: apps/src/templates/RegionalPartnerSearch.jsx [hunk 2]
```diff
@@ -137,6 +137,8 @@ class RegionalPartnerSearch extends Component {
 
     let workshopCollections = [
       {
+        key: 'CSD',
+        name: ActiveCourseWorkshops.CSD,
```
Comment (megcrenshaw) [inline apps/src/templates/RegionalPartnerSearch.jsx:158]: Love this approach!

File: apps/src/templates/RegionalPartnerSearch.jsx (hunk 1 above)
Comment (TurnerRiley) [reply apps/src/templates/RegionalPartnerSearch.jsx]: Good point! On it

**Pull request merged**